### PR TITLE
Add DO logo

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]

**Comment by juliusv** [comment]
The DO logo (the SVG file itself) looks completely cut off, also I'm not sure the `size="2"` makes sense? Also, there's still the word "Prometheus" after the DO logo, which I guess must be some copy-and-paste remnant?

File: content/community.html [hunk 1]
```diff
@@ -182,7 +182,7 @@ <h1>Acknowledgements</h1>
       <a href="https://www.docker.com/">Docker</a> and <a href="http://www.boxever.com/">Boxever</a>.
     </p>
     <p>
-      Special thanks to <a href="https://digitalocean.com/">DigitalOcean</a> for providing hosting resources.
+      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" width="200"></a> for providing hosting resources.
```
Comment (SuperQ) [inline content/community.html:185]: Width 200 takes up the whole page width, which is a bit much.

```suggestion
      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" width="50"></a> for providing hosting resources.
```

Comment (SuperQ) [reply content/community.html:185]: Actually, I'm not sure `width` works correctly with SVGs.

```suggestion
      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" style="width:200px"></a> for providing hosting resources.
```

File: content/community.html [hunk 2]
```diff
@@ -182,7 +182,7 @@ <h1>Acknowledgements</h1>
       <a href="https://www.docker.com/">Docker</a> and <a href="http://www.boxever.com/">Boxever</a>.
     </p>
     <p>
-      Special thanks to <a href="https://digitalocean.com/">DigitalOcean</a> for providing hosting resources.
+      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" style="width:200px"></a> for providing hosting resources.
```
Comment (SuperQ) [inline content/community.html:185]:
```suggestion
      Special thanks to DigitalOcean for providing hosting resources.
      <p>
        <a href="https://digitalocean.com/"><img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" style="width:200px;left:0"></a>
      </p>
```

**Comment by RichiH** [comment]
OK, after scaling down a raster image to a box of 200x200, it's still spanning the logo across the whole text block width. While that's not the default behaviour of HTML, at least I know that _something_ is messing with the image and have something to start from.

**Comment by RichiH** [comment]
image

**Comment by RichiH** [comment]
Line breaks around small images are not default behaviour either, but we're getting there 
image

**Comment by juliusv** [comment]
The raster image looks quite fuzzy, so an SVG would definitely be preferred. It's no problem to size SVGs using either image width/height attributes or CSS styling. The previous SVG image was just broken / cut off?

**Pull request merged**